manifests/05_role.yaml

```diff
@@ -26,7 +26,6 @@ rules:
   - operators.coreos.com
   resources:
   - catalogsources
-  - catalogsourceconfigs
   - operatorsources
```

Comment:
Can we remove the `operatorsources` resource as well? Given that we removed the OpSrc CRD from the manifests dir, there shouldn't be anything to query.

Reply:
Since the migrator needs to list the non-default operatorsources in the cluster ( (ref to this comment)[https://github.com/operator-framework/operator-marketplace/pull/323#discussion_r468836213] ) we need permission to list and delete the operatorsources.